packages/theming/android-theme/package.json

```diff
@@ -32,7 +32,8 @@
   "license": "MIT",
   "dependencies": {
     "@fluentui-react-native/theme-types": ">=0.18.0 <1.0.0",
-    "@fluentui-react-native/theme": ">=0.7.0 <1.0.0"
+    "@fluentui-react-native/theme": ">=0.7.0 <1.0.0",
+    "@fluentui-react-native/theme-tokens": "^0.19.0"
```

Comment:
FYI these packages are outdated now

Reply:
Also, running yarn test with this code fails some snapshot tests - would it safe to add all the updated snap files to fix those errors? Or there are additional recommended validations I should trigger?

Reply:
Updated the package versions and new snapshots